release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1459,6 +1459,8 @@ In the following tables, features are marked with the following statuses:
 
 * For a bonding network interface that holds a `br-ex` bridge device, do not set the `mode=6 balance-alb` bond mode in a node network configuration. This bond mode is not supported on {product-title} and it can cause the Open vSwitch (OVS) bridge device to disconnect from your networking environment. (link:https://issues.redhat.com/browse/OCPBUGS-34430[*OCPBUGS-34430*]).
 
+When a {product-title} {product-version} cluster is installed or configured as a post-installation activity on a single VIOS host with "vSCSI" storage on {ibm-power-name} with multipath configured, the CoreOS nodes with multipath enabled fail to boot. This behavior is expected as only one path is available to the node. (link:https://issues.redhat.com/browse/OCPBUGS-32290[*OCPBUGS-32290*]).
```

Comment:
Just needs a bullet before the note since all KI are in bullets. 
```suggestion
* When a {product-title} {product-version} cluster is installed or configured as a post-installation activity on a single VIOS host with "vSCSI" storage on {ibm-power-name} with multipath configured, the CoreOS nodes with multipath enabled fail to boot. This behavior is expected as only one path is available to the node. (link:https://issues.redhat.com/browse/OCPBUGS-32290[*OCPBUGS-32290*]).
```